### DISCUSSION TITLE
Error launch - python module and fbgem

### DISCUSSION THREAD

**Discussion by thecurang** [vladmandic/sdnext#3356] [Q&A]
I follow every step for installation and watch the videos too. But I always get the same error.

 D:\Aps\SDNEXT\launch.py:265 in <module>                                                                         │
│                                                                                                                      │
│   264 if __name__ == "__main__":                                                                                     │
│ ❱ 265     main()                                                                                                     │
│   266                                                                                                                │
│                                                                                                                      │
│ D:\Aps\SDNEXT\launch.py:242 in main                                                                             │
│                                                                                                                      │
│   241                                                                                                                │
│ ❱ 242     uv, instance = start_server(immediate=True, server=None)                                                   │
│   243     while True:                                                                                                │
│                                                                                                                      │
│ D:\Aps\SDNEXT\launch.py:168 in start_server                                                                     │
│                                                                                                                      │
│   167     get_custom_args()                                                                                          │
│ ❱ 168     module_spec.loader.exec_module(server)                                                                     │
│   169     uvicorn = None                                                                                             │
│ in exec_module:883                                                                                                   │
│ in _call_with_frames_removed:241                                                                                     │
│                                                                                                                      │
│ D:\Aps\SDNEXT\webui.py:11 in <module>                                                                           │
│                                                                                                                      │
│    10 from threading import Thread                                                                                   │
│ ❱  11 import modules.loader                                                                                          │
│    12 import torch # pylint: disable=wrong-import-order                                                              │
│                                                                                                                      │
│ D:\Aps\SDNEXT\modules\loader.py:14 in <module>                                                                  │
│                                                                                                                      │
│   13                                                                                                                 │
│ ❱ 14 import torch # pylint: disable=C0411                                                                            │
│   15 try:                                                                                                            │
│                                                                                                                      │
│ D:\Aps\SDNEXT\venv\lib\site-packages\torch\__init__.py:148 in <module>                                          │
│                                                                                                                      │
│    147                 err.strerror += f' Error loading "{dll}" or one of its dependencies.'                         │
│ ❱  148                 raise err                                                                                     │
│    149                                                                                                               │
╰──────────────────────────────────────────────────────────────────────────────────────────────────────────────────────╯
OSError: [WinError 126] The specified module could not be found. Error loading "D:\Aps\SDNEXT\venv\lib\site-packages\torch\lib\fbgemm.dll" or one of its dependencies.

**Comment by RexRomanas** [comment]
I get the same error at the same place in the SDNext install bat file. Windows 11 platform.

**Comment by RexRomanas** [comment]
This problem has been raised as an issue (https://github.com/vladmandic/automatic/issues/3360). The solution to install Visual Studio 2022 resolved the issue for me.

**Reply by Pojlekc** [reply]
What components did you add when installing Visual Studio 2022? I installed it without components. It didn't work.

**Reply by RexRomanas** [reply]
Per the advice in issue (https://github.com/vladmandic/automatic/issues/3360), I installed this option only.

![image](https://github.com/user-attachments/assets/4ccdcf96-726a-4f0b-a848-f50c9695e2b5)

**Reply by Pojlekc** [reply]
Thanks for the information! I already found the solution - I installed the dev version, it caused other errors but I was able to solve them.
